resource.go

```diff
@@ -18,48 +18,48 @@ package gowin32
 }
 
 var (
-	ResourceTypeCursor        ResourceType = wrappers.RT_CURSOR
-	ResourceTypeBitmap        ResourceType = wrappers.RT_BITMAP
-	ResourceTypeIcon          ResourceType = wrappers.RT_ICON
-	ResourceTypeMenu          ResourceType = wrappers.RT_MENU
-	ResourceTypeDialog        ResourceType = wrappers.RT_DIALOG
-	ResourceTypeString        ResourceType = wrappers.RT_STRING
-	ResourceTypeFontDir       ResourceType = wrappers.RT_FONTDIR
-	ResourceTypeFont          ResourceType = wrappers.RT_FONT
-	ResourceTypeAccelerator   ResourceType = wrappers.RT_ACCELERATOR
-	ResourceTypeRCData        ResourceType = wrappers.RT_RCDATA
-	ResourceTypeMessageTable  ResourceType = wrappers.RT_MESSAGETABLE
-	ResourceTypeGroupCursor   ResourceType = wrappers.RT_GROUP_CURSOR
-	ResourceTypeGroupIcon     ResourceType = wrappers.RT_GROUP_ICON
-	ResourceTypeVersion       ResourceType = wrappers.RT_VERSION
-	ResourceTypeDialogInclude ResourceType = wrappers.RT_DLGINCLUDE
-	ResourceTypePlugPlay      ResourceType = wrappers.RT_PLUGPLAY
-	ResourceTypeVxD           ResourceType = wrappers.RT_VXD
-	ResourceTypeAniCursor     ResourceType = wrappers.RT_ANICURSOR
-	ResourceTypeAniIcon       ResourceType = wrappers.RT_ANIICON
-	ResourceTypeHTML          ResourceType = wrappers.RT_HTML
-	ResourceTypeManifest      ResourceType = wrappers.RT_MANIFEST
+	ResourceTypeCursor        ResourceType = ResourceType(wrappers.RT_CURSOR)
+	ResourceTypeBitmap        ResourceType = ResourceType(wrappers.RT_BITMAP)
+	ResourceTypeIcon          ResourceType = ResourceType(wrappers.RT_ICON)
+	ResourceTypeMenu          ResourceType = ResourceType(wrappers.RT_MENU)
+	ResourceTypeDialog        ResourceType = ResourceType(wrappers.RT_DIALOG)
+	ResourceTypeString        ResourceType = ResourceType(wrappers.RT_STRING)
+	ResourceTypeFontDir       ResourceType = ResourceType(wrappers.RT_FONTDIR)
+	ResourceTypeFont          ResourceType = ResourceType(wrappers.RT_FONT)
+	ResourceTypeAccelerator   ResourceType = ResourceType(wrappers.RT_ACCELERATOR)
+	ResourceTypeRCData        ResourceType = ResourceType(wrappers.RT_RCDATA)
+	ResourceTypeMessageTable  ResourceType = ResourceType(wrappers.RT_MESSAGETABLE)
+	ResourceTypeGroupCursor   ResourceType = ResourceType(wrappers.RT_GROUP_CURSOR)
+	ResourceTypeGroupIcon     ResourceType = ResourceType(wrappers.RT_GROUP_ICON)
+	ResourceTypeVersion       ResourceType = ResourceType(wrappers.RT_VERSION)
+	ResourceTypeDialogInclude ResourceType = ResourceType(wrappers.RT_DLGINCLUDE)
+	ResourceTypePlugPlay      ResourceType = ResourceType(wrappers.RT_PLUGPLAY)
+	ResourceTypeVxD           ResourceType = ResourceType(wrappers.RT_VXD)
+	ResourceTypeAniCursor     ResourceType = ResourceType(wrappers.RT_ANICURSOR)
+	ResourceTypeAniIcon       ResourceType = ResourceType(wrappers.RT_ANIICON)
+	ResourceTypeHTML          ResourceType = ResourceType(wrappers.RT_HTML)
+	ResourceTypeManifest      ResourceType = ResourceType(wrappers.RT_MANIFEST)
```

Comment:
[Minor] With `MAKEINTRESOURCE` returning a `uintptr`, I'm pretty sure you don't need the explicit casts here.